docs/manual/01_Introduction.md

```diff
@@ -0,0 +1,52 @@
+# Introduction
+
+Clusterous is a tool for cluster computing on [AWS](https://aws.amazon.com). It lets you quickly and easily spin up a cluster of virtual machines on your AWS account and then deploy your parallel application in the form of [Docker](https://www.docker.com/what-docker) containers.
+
+The target users for Clusterous are scientists, data scientists and anyone else who needs on-demand access to powerful parallel computing resources for running their code. As such, it is designed to be easy to use for people who aren't system administrators or cloud computing experts. By simplifying many of the technical aspects of cloud computing, Clusterous lets you focus more on your algorithms and results, and less on APIs, networking and configuration.
+
+
+## Overview of operation
+
+Clusterous takes the form of a command-line tool, and is currently supported on Linux and Mac OS X (the clusters themselves run Linux). After installing Clusterous and configuring it with your AWS account credentials, you run the command to create a cluster, which is depicted in the following diagram.
+
```

Comment:
Any reason? I want to clarify to users that the server is always Linux, to avoid misconceptions that they can run any kind of software on the cluster.